core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -31,12 +31,21 @@ var ExplorationEditorTranslationTab = function() {
   this.exitTutorial = function() {
     // If the translation welcome modal shows up, exit it.
     translationWelcomeModal.isPresent().then(function(isVisible) {
-      if (isVisible) {
```

Comment:
My guess is that this was problematic because it doesn't wait at all for the modal to appear, so if the modal takes a while to appear, we won't exit it.

Does that match what you think? I'm trying to understand why this test was problematic so we don't repeat it!

Reply:
Yes, so the isVisible was false in most of my runs of this test. So it never entered the if block.